docker/README.md

```diff
@@ -0,0 +1,40 @@
+# Docker Action
```

Comment:
We are considering using GH's container registry, so we might not need this one. For the time being it's good to have it.